src/main/resources/db/migration/V1_6_Create_daily_list_table.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS moj_daily_list (
+  moj_dal_id                     INTEGER                     NOT NULL
```

Comment:
I think we should be adding the moj_dal_seq SEQUENCE to this script also with the table.

`CREATE SEQUENCE IF NOT EXISTS moj_dal_seq`